### DISCUSSION TITLE
D8F7ABD7-9AAC-1F7D-C1EC-EEDFEE26AF93

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#125] [General]
# Barnard's Star

Barnard's Star is a red dwarf about 5.96 light-years away in the constellation Ophiuchus. A candidate cold superterran planet was initially confirmed in 2018, but subsequent analyses in 2021 cast doubt about its existence

https://celestia.mobi/addon?item=D8F7ABD7-9AAC-1F7D-C1EC-EEDFEE26AF93

**Comment by GurrenLagannTSS** [comment]
Updating soon because i got to do the van de Kamp planets. Y'know, the ever-famous astrometric ones.

**Reply by Gliese710-max** [reply]
When?

**Comment by Gliese710-max** [comment]
This mod is clear and good 🔥
